docs/src/ncrings.md

```diff
@@ -259,6 +264,18 @@ Should returns `true` if parentheses are needed around this object when printed,
 a coefficient of a polynomial. As an example, non-constant polynomials would need such
 parentheses if used as coefficients of another polynomial.
 
+Note that since this approach quickly leads to unnecessary parentheses, the
+expression method below is preferred.
+
+### Expressions
+
+To obtain best results when printing composed types derived from other types, e.g., polynomials,
+the following method should be implemented.
+
+```julia
+expressify(f::MyElem; context = nothing)
+```
+
```

Comment:
Perhaps duplicate the rings version of this section here? Or alternatively, add something like
> For more information about `expressify`, please refer to the "Ring Interface" section"